rmw_opensplice_cpp/src/rmw_event.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014-2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dds_dcps.h>
+
+#include "rmw/allocators.h"
+#include "rmw/error_handling.h"
+#include "rmw/impl/cpp/macros.hpp"
+#include "rmw/event.h"
+#include "rmw/rmw.h"
+#include "rmw/types.h"
+
+#include "identifier.hpp"
+#include "types.hpp"
+
+extern "C"
+{
+/*
+ * Take an event from the event handle.
```

Comment:
Nitpick: add a doc brief (requested on the OSRF reviews, thought I'd be proactive here).